extensions/3DTILES_metadata/1.0.0/README.md

```diff
@@ -0,0 +1,1623 @@
+<!-- omit in toc -->
+# 3DTILES_metadata Extension
+
+**Version 1.0.0**, February 25, 2021
+
+<!-- omit in toc -->
+## Contributors
+
+* Sean Lilley, Cesium
+* Peter Gagliardi, Cesium
+* Sam Suhag, Cesium
+* Patrick Cozzi, Cesium
```

Comment:
Please put me last in any new extensions that I contributed to in a material way.